source/ArrowKeyStepper/ArrowKeyStepper.js

```diff
@@ -14,11 +14,12 @@ export default class ArrowKeyStepper extends PureComponent {
   };
 
   static propTypes = {
-    children: PropTypes.func.isRequired,
+    children: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
```

Comment:
This change seems wrong. If you pass a node, wouldn't things break now since the `render` function expect `children` to always be a function?